framework/src/modules/chain/scripts/console.js

```diff
@@ -61,7 +61,7 @@ application.init(
 		const helpersFolder = './helpers/';
 		fs.readdirSync(helpersFolder).forEach(file => {
 			const filePath = path.resolve(helpersFolder, file);
-			const fileName = path.basename(filePath, '.js');
+			const fileName = path.basename(filePath, '');
```

Comment:
I think we shouldn't remove .js from here fileName was being used as a key in the helpers object. Can cause a side effect.

But if you don't see a problem, we can remove fileName variable and useFilePath as a key.